### PR TITLE
the project now has grunt,npm and scss.

### PR DESCRIPTION
**Pull request by douglasdeodato**
- Feautures Implemented
- Grunt and npm installed.
- Grunt tasks installed:  "load-grunt-tasks","time-grunt","grunt-contrib-sass","grunt-contrib-watch","grunt-contrib-connect".
- we can run a server with  the command line grunt serve: http://localhost:8000
- we can compile the scss  for distributio and development with grunt sass
- we can be watching scss and other changes in the app with grunt watch
- SCSS is ready for development, I will start in the next PR.

### CODE REVIEW

**Comment by douglasdeodato** [comment]
hey guys and @NamPNQ , when you will accept my PR? thank you.

**Comment by stevemao** [comment]
Thanks for the PR! Personally I'd prefer to use `gulp` or `npm script`, for obvious reason :) or none just to make this as simple as possible. But I'll leave it to other contributors 😄

**Comment by douglasdeodato** [comment]
@stevemao  we could have both , I created the grunt one, because was none and I will start make a sass version of the css files as I discussed here: https://github.com/you-dont-need/You-Dont-Need-Javascript/issues/61 , anyway I am trying to collaborate with the project, soon this PR accepted I can keep going on the SCSS version of this project.

**Comment by NamPNQ** [comment]
@douglasdeodato Sorry for I missed this

Great if you add an sample(ex: https://github.com/you-dont-need/You-Dont-Need-Javascript/blob/master/examples/carousel-1.html) and update contribute guide

**Comment by NamPNQ** [comment]
@stevemao I think `gulp` or `grunt` not is a big problem

**Comment by cht8687** [comment]
@NamPNQ @stevemao  so are we merging this PR?

**Comment by nufrifin** [comment]
But it's javascript

**Comment by DanielRuf** [comment]
Pure npm scripts would be better.

**Comment by brodybits** [comment]
> Pure npm scripts would be better.

What do you mean by "pure"?

**Comment by DanielRuf** [comment]
Without grunt, gulp and so on.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "url": "git+https://github.com/you-dont-need/You-Dont-Need-Javascript"
+  },
+  "keywords": [
+    "you",
+    "dont",
+    "need",
+    "javascript"
+  ],
+  "author": "you-dont-need",
+  "license": "ISC",
```
Comment (brodybits) [inline package.json]: Not consistent with [existing license](https://github.com/you-dont-need/You-Dont-Need-JavaScript/blob/master/LICENSE)

Comment (DanielRuf) [reply package.json]: It's the default of `npm init` (MIT compatible?)

Comment (brodybits) [reply package.json]: Yes MIT compatible but not the same as the existing project license (WTFPL). I would favor seeking consensus before marking this project with a different license.

**Pull request closed without merging**